packages/bpk-mixins/src/mixins/_typography.scss

```diff
@@ -32,9 +32,12 @@
 ///
 /// @access private
 
-@mixin _bpk-text-factory($font-size, $line-height, $font-weight: false, $letter-spacing: false) {
+@mixin _bpk-text-factory($font-size, $font-weight: false, $line-height: false, $letter-spacing: false) {
```

Comment:
Swapped these around as was testing the effects or removing line-heights and letter-spacing.

Also made line-height optional as the new larger sizes don't have line-heights due to the font

Reply:
Can we remove the two optional parameters since they are not used.

Reply:
We should also include in the comms for this that you have to update the base stylesheet to get the correct line-height